spec/requests/api/v1/auth/passwords/create_spec.rb

```diff
@@ -0,0 +1,21 @@
+describe 'POST api/v1/auth/passwords', type: :request do
+  let(:user) { create(:user) }
+  let(:params) { { email: user.email } }
+
+  subject do
+    post api_v1_user_password_path, params: params, as: :json
+    response
+  end
+
+  context 'when user exists' do
```

Comment:
use full sentences, dont omit the article: `when THE user exists`. 
Or better: `when there is no user by that email`. Same below